packages/core/src/connection/transactional-connection.ts

```diff
@@ -10,7 +11,9 @@ import {
     ObjectLiteral,
     ObjectType,
     Repository,
+    SelectQueryBuilder,
 } from 'typeorm';
+import { DataSource } from 'typeorm/data-source/DataSource';
```

Comment:
should this (and the `InjectDataSource` above) really be deep imports?

Reply:
ok, I will fix it. this is from my IDE, I didn't noticed

Reply:
done

> should this (and the `InjectDataSource` above) really be deep imports?